docs/release/release_0_5_0.md

```diff
@@ -0,0 +1,911 @@
+- fix `create_func` to respect `rename_argument` decorator ([napari/napari#6970](https://github.com/napari/napari/pull/6970))
+- Do not crash napari viewer if error happen on open file via cli ([napari/napari#6971](https://github.com/napari/napari/pull/6971))
+- Fix viewer scale bar color update ([napari/napari#6989](https://github.com/napari/napari/pull/6989))
+- Remove NumPy 2 pin, fix direct colormap with keys outside data dtype ([napari/napari#6998](https://github.com/napari/napari/pull/6998))
+- Fix switching from 2d to 3d with 2d points ([napari/napari#7004](https://github.com/napari/napari/pull/7004))
+- Fix importerror and message for lxml_html_clean ([napari/napari#7017](https://github.com/napari/napari/pull/7017))
+
+## API Changes
+
+- Add .to_rgb_dict() to Theme class. ([napari/napari#4759](https://github.com/napari/napari/pull/4759))
+- Thick Slices (Dims  as nD-box instead of nD-point) ([napari/napari#5522](https://github.com/napari/napari/pull/5522))
+- Lock dimensions / axes while rolling ([napari/napari#5986](https://github.com/napari/napari/pull/5986))
+- Remove deprecated code from Labels layer ([napari/napari#6641](https://github.com/napari/napari/pull/6641))
+
+## Deprecations
+
+- MAINT: Remove unused function. ([napari/napari#5041](https://github.com/napari/napari/pull/5041))
+- MAINT: remove deprecated Alias. ([napari/napari#5043](https://github.com/napari/napari/pull/5043))
+- Fix typing in _app_model ([napari/napari#6059](https://github.com/napari/napari/pull/6059))
+- Rename `edge_*` attributes and references to `border_*` on points layer ([napari/napari#6402](https://github.com/napari/napari/pull/6402))
+- Add deprecation message to CallDefault function partly resolves issue#6257 ([napari/napari#6901](https://github.com/napari/napari/pull/6901))
+
+## Build Tools
+
+- ci(dependabot): bump WebFreak001/deploy-nightly from 1.1.0 to 1.2.0 ([napari/napari#5159](https://github.com/napari/napari/pull/5159))
+- ci(dependabot): bump convictional/trigger-workflow-and-wait from 1.6.1 to 1.6.4 ([napari/napari#5293](https://github.com/napari/napari/pull/5293))
+- ci(dependabot): bump bruceadams/get-release from 1.2.3 to 1.3.2 ([napari/napari#5294](https://github.com/napari/napari/pull/5294))
+- ci(dependabot): bump convictional/trigger-workflow-and-wait from 1.6.4 to 1.6.5 ([napari/napari#5384](https://github.com/napari/napari/pull/5384))
+- ci(dependabot): bump WebFreak001/deploy-nightly from 1.2.0 to 2.0.0 ([napari/napari#5385](https://github.com/napari/napari/pull/5385))
+- ci(dependabot): bump docker/build-push-action from 3 to 4 ([napari/napari#5523](https://github.com/napari/napari/pull/5523))
+- ci(dependabot): bump toshimaru/auto-author-assign from 1.6.1 to 1.6.2 ([napari/napari#5524](https://github.com/napari/napari/pull/5524))
+- fix various keybinding/shortcut bugs ([napari/napari#5604](https://github.com/napari/napari/pull/5604))
```

Comment:
weird one here

Reply:
The labels:
<img width="343" alt="image" src="https://github.com/napari/docs/assets/76622105/b5cb6282-fd04-4318-a4c0-1ea1d022891c">

I guess dependencies puts it here?
Maybe tag it with `bugfix`? that seems what it does based on the first post.

Reply:
yup, sounds good!